man/man5/zfs-module-parameters.5

```diff
@@ -1765,6 +1765,20 @@ table.
 Default value: \fB131,072\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBfletcher_4_impl_set\fR (string)
```

Comment:
**fletcher_4_impl_set** should be changed to *zfs_fletcher_4_impl_set**.  This entire section also needs to be moved up so it appears alphabetically in the correct order.